src/server/src/test/java/io/cassandrareaper/acceptance/ReaperTestJettyRunner.java

```diff
@@ -51,10 +52,16 @@
 
   static final String REAPER_APPLICATION_FQN = "io.cassandrareaper.ReaperApplication";
 
+  public final DatacenterAvailability datacenterAvailability;
+
   final ReaperJettyTestSupport runnerInstance;
   private SimpleReaperClient reaperClientInstance;
 
-  public ReaperTestJettyRunner(String yamlConfigFile, Optional<String> version) {
+  public ReaperTestJettyRunner(
+      String yamlConfigFile,
+      Optional<String> version,
+      Optional<DatacenterAvailability> dcAvailability) {
```

Comment:
i don't think this added parameter is needed. The `DatacenterAvailability` is already specified in the yaml.

Reply:
Good point, I hadn't noticed I could do this.